templates/patterns/card/pattern-card--variant-search.html.twig

```diff
@@ -54,7 +54,6 @@
   'text_color': 'dark',
   'image': (image is not empty) ? image|merge({
     'position': 'top',
-    'classes': 'd-none d-md-block',
```

Comment:
I never understood why the image needs to disappear in mobile view, but now that we change, is this a bug fix or a change request?